rust/src/tx_builder.rs

```diff
@@ -342,6 +342,7 @@ pub struct TransactionBuilder {
     required_signers: Ed25519KeyHashes,
     collateral_return: Option<TransactionOutput>,
     total_collateral: Option<Coin>,
+    reference_inputs: TransactionInputs,
```

Comment:
@vsubhuman Do we need make it Optional ?

Reply:
@lisicky , no need